### ISSUE TITLE
[BUG] Turret sticking / critical lag

### ISSUE THREAD

**Issue by cloudrambler** [tsunamayo/Starship-EVO#3558]
### Short Description of the issue:

A turret that I have been working on for a while has a persistent problem where it will not move under manual control. Occasionally it moves fine the first time I use it, but mostly it moves for a fraction of a second and then freezes. This repeats each time I engage the turret manually. Also sometimes it will move, then stop, then every few seconds there is a brief period where it will move again if the mouse if moving.

Initially the turret was quite heavy, which was causing other problems, so I reduced the mass to where it should always work, but this bug is stopping me progress the design. I have tried it in a fresh world immediately after loading the game and it is still not working.

Since you have just updated the turrets and the problem remains, I figured it is a good time to shout.

### Steps to reproduce the issue:

Load the attached blueprint of a concept ship (its messed up due to copy/paste errors).
Try to use the turret manually.

### Environment
- Starship EVO version: 21w04b
- Graphics Card: GTX1070

### Additional information:

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/5897824/Player.log)
[ASSAULT FRIGATE 4.zip](https://github.com/tsunamayo/Starship-EVO/files/5897826/ASSAULT.FRIGATE.4.zip)

**Comment by cloudrambler** [comment]
Also I should point out that this particular turret can get physically stuck. It can move to positions where it significantly clips with itself and then cannot move. Plus the new collision map is not correct for this turret, as it shows targeting vectors that the second axis is physically incapable of aligning to.

**Comment by tsunamayo** [comment]
Hi, does this turret have anything special in it?

**Comment by cloudrambler** [comment]
@tsunamayo I cannot think of anything specific. It's not very old as I am still working on the concept for the ship with these large integrated turrets. I did have more lasers to start with, but removed them (hopefully temporarily) as the mass was causing other movement problems. I also noticed that the hinge (second axis) has some spurious angular position; which may be confusing things. But otherwise I think it's fairly standard.
